server/app/controllers/taskActions.js

```diff
@@ -0,0 +1,22 @@
+const tables = require("../../database/tables");
+
+const browse = async ({ res, next }) => {
+    try {
+        const [tasks] = await tables.task.getAll();
+        res.status(200).json(tasks);
+    } catch (err) {
+        next(err);
+    }
+};
+
+const getByStatus = async (req, res, next) => {
+    try {
+        const status = req.body;
```

Comment:
use destructuration  to get the status from the params object : const { status } = req.params

Reply:
done

Reply:
done